src-tests/Process/ProcessSetCreatorTest.php

```diff
@@ -276,11 +280,39 @@ public function testShouldPropagateCustomOptionsIntoProcess()
                 'PUBLISH_RESULTS' => '1',
                 'FIXTURES_DIR' => 'custom-fixtures-dir/',
                 'LOGS_DIR' => 'custom-logs-dir/',
+                'CAPABILITY' => '{"webdriver.log.file":"\/foo\/bar.log","capability.in.quotes":"\/foo\/ba r.log",'
+                    . '"platform":"OS X 10.8"}',
             ],
             $processEnv
         );
     }
 
+    public function testShouldNotAcceptCapabilitiesInWrongFormat()
+    {
+        $this->input = new StringInput(
+            'foo chrome'
+            . ' --' . RunCommand::OPTION_CAPABILITY . '=foo'
+        );
+
+        $this->input->bind($this->command->getDefinition());
+
+        // Redeclare creator so it uses the new input
+        $this->creator = new ProcessSetCreator(
+            $this->command,
+            $this->input,
+            $this->bufferedOutput,
+            $this->publisherMock
+        );
+
+        $files = $this->findDummyTests('DummyTest.php'); // find only one file (we don't need more for the test)
```

Comment:
Maybe `$file`? Or add extra method for searching only one file to avoid extra comment next to each usage.